zerberus/zerberus.cpp

```diff
@@ -257,7 +257,7 @@ void Zerberus::updatePatchList()
       patches.clear();
       int idx = 0;
       for (ZInstrument* i : instruments) {
-            Ms::MidiPatch* p = new Ms::MidiPatch { false, name(), 0, idx, i->name() };
+            Ms::MidiPatch* p = new Ms::MidiPatch { false, name(), 0, idx, 0, i->name() };
```

Comment:
@jthistle what is the purpose of `sfid`? Why do you use `0` value here? Aren't enum values better to specify zerberus/fluid types?

Reply:
sfid isn't an enum-able value. It's used only in fluid, because fluid iterates over each sf separately, so it's used to distinguish patches from different soundfonts from each other, thus allowing patches to be displayed in the correct order in the mixer. Zerberus, AFAIK, just iterates over every patch regardless of soundfont, so the 0 is just a placeholder, really.